[BUG] Windows version of 0.19 throw an error on dagger up

I have two environments:
-WSL
-Windows 10
I created an empty workspace (no .cue files), did dagger init and dagger new 'myworkplace'

Expected result:
```bash
user@laptop:/mnt/c/Users/user/.alefesta/dg-orig$ dagger up
1:46PM FTL system | failed to up environment: plan config: no CUE files in .
```

on WSL this works fine (Linux binary) on Windows I get:
```powershell
panic: illegal character U+0073 's' in escape sequence:
    1:28


goroutine 44 [running]:
go.dagger.io/dagger/environment.(*Environment).LocalDirs(0xc000610560, 
0x1f349a0)
        /home/runner/work/dagger/dagger/environment/environment.go:174 
+0x3e5
go.dagger.io/dagger/client.(*Client).buildfn(0xc000286630, 0x237f378, 0xc00046e600, 0xc000545810, 0xc000610560, 0xc0002aa378, 0xc00015a840, 0xc000396d20, 0xc0000c20e0)
        /home/runner/work/dagger/dagger/client/client.go:102 +0xa6     
go.dagger.io/dagger/client.(*Client).Do.func2(0xc000396ee0, 0xc0000c20e0)
        /home/runner/work/dagger/dagger/client/client.go:92 +0x67      
golang.org/x/sync/errgroup.(*Group).Go.func1(0xc0001b4270, 0xc00046e780)
        /home/runner/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57 +0x62
created by golang.org/x/sync/errgroup.(*Group).Go
        /home/runner/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:54 +0x71
```

as per the image attached
![dagger-bug](https://user-images.githubusercontent.com/80312181/125617814-d7838527-67ee-4a3f-8d53-fd11886cae31.JPG)

--- a/state/input.go
+++ b/state/input.go
@@ -119,9 +119,14 @@ func (dir dirInput) Compile(_ string, state *State) (*compiler.Value, error) {
 		return nil, fmt.Errorf("%q dir doesn't exist", dir.Path)
 	}
 
+	dirPath, err := json.Marshal(p)
+	if err != nil {
+		return nil, err
+	}
+
 	llb := fmt.Sprintf(
-		`#up: [{do:"local",dir:"%s", include:%s, exclude:%s}]`,
-		p,
+		`#up: [{do:"local",dir:%s, include:%s, exclude:%s}]`,
+		dirPath,
 		includeLLB,
 		excludeLLB,
 	)

